README.md

```diff
@@ -118,11 +112,15 @@
 * **[CAMEL](https://github.com/camel-ai/camel)**: 🐫 CAMEL: The first and the best multi-agent framework. Finding the Scaling Law of Agents. ![Stars](https://img.shields.io/github/stars/camel-ai/camel.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/camel-ai/camel?color=green) ![LastCommit](https://img.shields.io/github/last-commit/camel-ai/camel?color=green)
 * **[Codex](https://github.com/openai/codex)**: Lightweight coding agent that runs in your terminal ![Stars](https://img.shields.io/github/stars/openai/codex.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/openai/codex?color=green) ![LastCommit](https://img.shields.io/github/last-commit/openai/codex?color=green) ![Tag](https://img.shields.io/badge/coding-orange)
 * **[kagent](https://github.com/kagent-dev/kagent)**: kagent is a kubernetes native framework for building AI agents. ![Stars](https://img.shields.io/github/stars/kagent-dev/kagent.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/kagent-dev/kagent?color=green) ![LastCommit](https://img.shields.io/github/last-commit/kagent-dev/kagent?color=green) ![Tag](https://img.shields.io/badge/kubernetes-orange)
+* **[LangChain](https://github.com/langchain-ai/langchain)**: 🦜🔗 Build context-aware reasoning applications. ![Stars](https://img.shields.io/github/stars/langchain-ai/langchain.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/langchain-ai/langchain?color=green)  ![LastCommit](https://img.shields.io/github/last-commit/langchain-ai/langchain?color=green)
 * **[LangGraph](https://github.com/langchain-ai/langgraph)**: Build resilient language agents as graphs. ![Stars](https://img.shields.io/github/stars/langchain-ai/langgraph.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/langchain-ai/langgraph?color=green) ![LastCommit](https://img.shields.io/github/last-commit/langchain-ai/langgraph?color=green)
+* **[LightRAG](https://github.com/HKUDS/LightRAG)**: "LightRAG: Simple and Fast Retrieval-Augmented Generation" ![Stars](https://img.shields.io/github/stars/HKUDS/LightRAG.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/HKUDS/LightRAG?color=green)  ![LastCommit](https://img.shields.io/github/last-commit/HKUDS/LightRAG?color=green)
```

Comment:
I personally think this should be part of the agent, cc @samzong for opinions as well.

Reply:
move to Agent is a better choice, LightRAG is designed to provide Web and API built service, It's need base on Open AI-like or other models.